types/index.d.ts

```diff
@@ -3149,6 +3149,7 @@ declare namespace Knex {
     loadExtensions?: readonly string[];
     migrationSource?: MigrationSource<unknown>;
     name?: string;
+    getNewMigrationName?: (name: string) => string;
```

Comment:
I'm not sure this is accurate. it's not a part of migrator config, to my knowledge